### ISSUE TITLE
Source Switcher doesn't work on Edge 16

### ISSUE THREAD

**Issue by gibigate** [fluid-player/fluid-player#185]
i think problem with this part of player
```
player.videoSources.forEach(function(source) {
                        if (source.title == videoChangedTo.innerText) {
                            player.setBuffering();
                            player.setVideoSource(source.url);
                        }
                    });
```

**Comment by cian-exads** [comment]
I can reproduce the issue @gibigate, marked this as a bug to be looked into.

**Comment by cian-exads** [comment]
Hi @gibigate, your fix for this issue went live with v2.4.0. Thanks for your contribution! Closing issue.
